Resources/doc/suggesters/usage.rst

```diff
@@ -0,0 +1,87 @@
+Suggesters Usage
+================
+
+`Suggesters`_ are used to find a similar looking term using the provided text.
+
+You can use suggesters in any query as you would normally, or `suggester api`_.
+
+There are basically four types of suggesters: terms, phrase, completion and context, to find out available parameters for these suggesters check out `<types.rst>`_ section.
```

Comment:
There are basically four types of suggesters: terms, phrase, completion and context. To find out available parameters for these suggesters check out `<types.rst>`_ section.